src/main/kotlin/calculator/Formula.kt

```diff
@@ -0,0 +1,40 @@
+package calculator
+
+class Formula(formula: String?) {
+
+    private lateinit var operand: List<Operand>
+
+    init {
+        makeFormula(formula ?: "0")
+    }
+
+    fun calculate(): Int {
+        return operand.sumOf { it.getValue() }
+    }
+
+    private fun makeFormula(formula: String) {
+        if (makeZeroWhenIsBlank(formula)) return
+        val result = customDelimiterRegex.find(formula)
+
+        result?.let { matchResult ->
+            val customDelimiter = matchResult.groupValues[1]
+            operand = matchResult.groupValues[2].split(customDelimiter).map { Operand(it) }
+            return
+        }
+        operand = formula.split(defaultDelimiter).map { Operand(it) }
```

Comment:
```suggestion
        operand = result?.let { matchResult ->
            val customDelimiter = matchResult.groupValues[1]
            matchResult.groupValues[2].split(customDelimiter).map { Operand(it) }
        } ?: formula.split(defaultDelimiter).map { Operand(it) }  
```

`?:`을 사용하면 return을 지울 수 있어보이네요!